src/collection-view.js

```diff
@@ -76,7 +76,7 @@ const CollectionView = Backbone.View.extend({
   },
 
   _endBuffering() {
-    const shouldTriggerAttach = !!this._isAttached;
+    const shouldTriggerAttach = !!this._isAttached && this.monitorViewEvents !== false;
```

Comment:
My point was rather `!!foo` => `!!foo && bool` could be written `foo && bool` because of the `&&` making the cast not because `foo` might be already a boolean